kubernetes/linux/Dockerfile

```diff
@@ -3,9 +3,10 @@ MAINTAINER OMSContainers@microsoft.com
 LABEL vendor=Microsoft\ Corp \
     com.microsoft.product="OMS Container Docker Provider" \
     com.microsoft.version="10.0.0-1"
+ARG IMAGE_TAG=ciprod05262020
+ENV AGENT_VERSION ${IMAGE_TAG}

```

Comment:
i see the same in windows docker file as well. Since we cannot build 2 images with same tag, can you append "win-" prefix as convention to windows ? or else have 2 env image_tag_linux & image_tag_windows may be ?

Reply:
> i see the same in windows docker file as well. Since we cannot build 2 images with same tag, can you append "win-" prefix as convention to windows ? or else have 2 env image_tag_linux & image_tag_windows may be ?

> this is covered as readme to specify the image tag as part of the build command.

Reply:
> > i see the same in windows docker file as well. Since we cannot build 2 images with same tag, can you append "win-" prefix as convention to windows ? or else have 2 env image_tag_linux & image_tag_windows may be ?
> 
> > this is covered as readme to specify the image tag as part of the build command.

Fixed.